pkgs/stdenv/darwin/default.nix

```diff
@@ -190,13 +190,10 @@ in rec {
         useSharedLibraries = false;
       };
 
-      python = super.callPackage ../../development/interpreters/python/cpython/2.7/boot.nix {
-        CF = null;  # use CoreFoundation from bootstrap-tools
-        configd = null;
-      };
-      python2 = self.python;
+      python3 = super.python3Minimal;
```

Comment:
@FRidh This used to be a separate (minimal) copy to make python updates easier, would you like me to do the same for python3?

Reply:
Darwin has `pkgs/development/interpreters/python/cpython/2.7/boot.nix` which is now removed. If you think you need that for Python 3, then yes, you can create it. Maybe you can indeed use `python3Minimal`? Note that was added by @adisbladis and serves a different purpose.

Reply:
Strictly speaking it's not necessary. But otherwise changing anything anything that's not version guarded, like the setup-hook, will also change this. Causing a full stdenv rebuild which can be pretty awkward.

Reply:
It's not often we change the Python 3 expression, and when we do it goes to staging. Anyway, it's up to you.

Reply:
Yeah the python2 boot thing was more of a hack than anything. I actually thought it had to do with needing configd for cpython, which looks like is no longer necessary.

Reply:
I'll leave as is for now, we can always add that later if it's a problem.